201-aci-wordpress-vnet/azuredeploy.json

```diff
@@ -0,0 +1,484 @@
+      "metadata": {
+        "description": "Subnet name for ACI"
+      }
+    },
+    "subnet2AddressPrefix": {
+      "type": "string",
+      "defaultValue": "10.0.1.0/24",
+      "metadata": {
+        "description": "Subnet prefix for application gateway"
+      }
+    },
+    "subnet2Name": {
+      "type": "string",
+      "defaultValue": "ag-subnet",
+      "metadata": {
+        "description": "Subnet name for application gateway"
+      }
+    },
+    "mysqlPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "MySQL database password"
+      }
+    },
+    "location": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().location]",
+      "metadata": {
+        "description": "Location for all resources."
+      }
+    }
+  },
+  "variables": {
+    "storageAccountName": "[uniquestring(resourceGroup().id)]",
+    "storageAccountType": "Standard_LRS",
+    "publicIPAddressName": "publicIp1",
+    "publicIPRef": "[resourceId('Microsoft.Network/publicIPAddresses',variables('publicIPAddressName'))]",
+    "networkProfileName": "aci-networkProfile",
+    "interfaceConfigName": "eth0",
+    "interfaceIpConfig": "ipconfigprofile1",
+    "image": "microsoft/azure-cli",
+    "shareContainerGroupName": "createshare-containerinstance",
+    "wordpressContainerGroupName": "wordpress-containerinstance",
+    "mysqlContainerGroupName": "mysql-containerinstance",
+    "wordpressShareName": "wordpress-share",
+    "mysqlShareName": "mysql-share",
+    "port": 80,
+    "cpuCores": "1.0",
+    "memoryInGb": "1.5",
+    "skuName": "Standard_Medium",
+    "capacity": "2",
+    "applicationGatewayName": "applicationGateway1",
+    "subnet2Ref": "[resourceId('Microsoft.Network/virtualNetworks/subnets', parameters('vnetName'), parameters('subnet2Name'))]",
+    "applicationGatewayID": "[resourceId('Microsoft.Network/applicationGateways',variables('applicationGatewayName'))]",
+    "wordpressContainerGroupRef": "[resourceId('Microsoft.ContainerInstance/containerGroups/', variables('wordpresscontainerGroupName'))]",
+    "mysqlContainerGroupRef": "[resourceId('Microsoft.ContainerInstance/containerGroups/', variables('mysqlContainerGroupName'))]",
+    "siteName": "aciwordpresssite"
+  },
+  "resources": [
+    {
+      "type": "Microsoft.Storage/storageAccounts",
+      "name": "[variables('storageAccountName')]",
+      "apiVersion": "2017-10-01",
+      "location": "[parameters('location')]",
+      "sku": {
+        "name": "[variables('storageAccountType')]"
+      },
+      "kind": "Storage",
+      "properties": { }
+    },
+    {
+      "apiVersion": "2015-05-01-preview",
+      "type": "Microsoft.Network/publicIPAddresses",
+      "name": "[variables('publicIPAddressName')]",
+      "location": "[parameters('location')]",
+      "properties": {
+        "publicIPAllocationMethod": "Dynamic",
+         "dnsSettings": {
+          "domainNameLabel": "[variables('siteName')]"

```

Comment:
this needs to be globally unique - you'll need to generate something with uniqueString()